example/src/InfoPane/InfoPane.css

```
@@ -0,0 +1,17 @@
+.info-pane {
+    z-index: 9;
+    background-color: white;
+    padding: 16px;
+    box-shadow: 0px 1px 1px -1px RGB(0, 0, 0, 0.12), 0px 1px 1px 0px RGB(0, 0, 0, 0.14), 0px 1px 3px 0px RGB(0, 0, 0, 0.12);
```

Stylelint (17.14.0)

[error] Expected "RGB" to be "rgb" (function-name-case)

(function-name-case)